scripts/gen_operator_csv.py

```diff
@@ -64,7 +64,7 @@
     print("Generating CSV for version: %s" % operator_version)
 
     with open('scripts/templates/csv.yaml', 'r') as stream:
-        csv = yaml.load(stream, Loader=yaml.FullLoader)
+        csv = yaml.load(stream, Loader=yaml.BaseLoader)
```

Comment:
Tested that using `make catalog` and it's adding some garbage to the output file (python!unicode characters) that are making the catalog container crash during startup. The suggestion is going with:
```python
csv = yaml.load(stream)
```
Tested locally and the output is clean.